packages/flutter/lib/src/cassowary/equation_member.dart

```diff
@@ -5,25 +5,64 @@
 import 'expression.dart';
 import 'constraint.dart';
 
-/// Base class for the various parts of cassowary equations.
+/// Represents a member that can be used to construct an expression that may be
+/// used to create a constraint. This is to facilitate the easy creation of
+/// constraints. The use of the operator overloads is completely optional and
+/// is only meant as a convenience. The constraint expressions can be created
+/// by manually creating instance of constraint variables, then terms and
+/// combining those to create expression.
 abstract class EquationMember {
+  /// The representation of this member after it is hoisted to be an
+  /// expression.
   Expression asExpression();
 
+  /// If this member is a constant.
```

Comment:
too obvious, add colour. e.g. what does it mean for it to be a constant? What are the implications? Why would you use this?